src/core/Akka.Cluster/Gossip.cs

```diff
@@ -71,10 +72,12 @@ public static Gossip Create(ImmutableSortedSet<Member> members)
         readonly ImmutableSortedSet<Member> _members;
         readonly GossipOverview _overview;
         readonly VectorClock _version;
+        private readonly Lazy<Reachability> _reachability;
 
         public ImmutableSortedSet<Member> Members { get { return _members; } }
         public GossipOverview Overview { get { return _overview; } }
         public VectorClock Version { get { return _version; } }
+        public Reachability ReachabilityExcludingDownedObservers { get { return _reachability.Value; } }
```

Comment:
Probably a placeholder. I copied it over from akka. Check out the lazy function to see what it does.